firestore/noxfile.py

```diff
@@ -139,22 +138,3 @@ def cover(session):
     session.run("coverage", "report", "--show-missing", "--fail-under=100")
 
     session.run("coverage", "erase")
-
-@nox.session(python="3.7")
-def docs(session):
-    """Build the docs for this library."""
-
-    session.install('-e', '.')
-    session.install('sphinx', 'alabaster', 'recommonmark')
-
-    shutil.rmtree(os.path.join('docs', '_build'), ignore_errors=True)
-    session.run(
-        'sphinx-build',
-        '-W',  # warnings as errors
-        '-T',  # show full traceback on exception
-        '-N',  # no colors
-        '-b', 'html',
-        '-d', os.path.join('docs', '_build', 'doctrees', ''),
-        os.path.join('docs', ''),
-        os.path.join('docs', '_build', 'html', ''),
-    )
```

Comment:
Does the docs session need to be removed as well?

Reply:
I would prefer not:  that seems to be some kind of wonk in the templating.

Reply:
I have regenerated / rebased to pick up the new `noxfile.py` template.